### PR TITLE
Announcement about Intel sponsorship

### PR DESCRIPTION
**Pull request by apawlik**

### CODE REVIEW

File: blog/2015/11/intel-sponsor-wise-uk.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<!-- start excerpt -->
+<p>We are very happy to announce that the first <a href="http://apawlik.github.io/2015-12-wise-uk/">
+UK Software Carpentry Workshop for Women in Science and Engineering</a> 
+received generous support from <a href="http://www.intel.co.uk/">Intel</a>. 
+The event will take place on 14th and 15th December at the University of Manchester.
+</p>
+<!-- end excerpt -->
+<div style="text-align: center">
+<a href="http://www.intel.co.uk/">
+<img src="http://apawlik.github.io/2015-12-wise-uk/img/intel-logo.jpg" alt="Intel logo" style="height:120px"/></a>
```
Comment (gvwilson) [inline blog/2015/11/intel-sponsor-wise-uk.html]: Please copy the image into the blog/2015/11 directory and refer to the local copy.

File: blog/2015/11/intel-sponsor-wise-uk.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+</div>
+<p>We are very grateful for Intel's support, and look forward to the event!&nbsp;</p>
```
Comment (gvwilson) [inline blog/2015/11/intel-sponsor-wise-uk.html]: Why the non-breaking space at the end?

File: blog/2015/11/intel-sponsor-wise-uk.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+title: "Intel to sponsor the first Workshop for Women in Science and Engineering in the UK"
+date: 2015-11-25
+time: "20:00:00"
+category: ["WiSE", "Women in Science and Engineering"]
+---
+<!-- start excerpt -->
+<p>We are very happy to announce that the first <a href="http://apawlik.github.io/2015-12-wise-uk/">
+UK Software Carpentry Workshop for Women in Science and Engineering</a> 
+received generous support from <a href="http://www.intel.co.uk/">Intel</a>. 
+The event will take place on 14th and 15th December at the University of Manchester.
```
Comment (gvwilson) [inline blog/2015/11/intel-sponsor-wise-uk.html]: "on December 14th and 15"

File: blog/2015/11/intel-sponsor-wise-uk.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+layout: blog
+root: ../../..
+author: Aleksandra Pawlik
+title: "Intel to sponsor the first Workshop for Women in Science and Engineering in the UK"
+date: 2015-11-25
+time: "20:00:00"
```
Comment (gvwilson) [inline blog/2015/11/intel-sponsor-wise-uk.html]: Please change to 12:00 (since the server that hosts this is on Pacific time).

**Comment by gvwilson** [comment]
@apawlik @jduckles do we want to merge/publish this one today?

**Comment by jduckles** [comment]
@apawlik Just waiting on your changes suggested by @gvwilson, or do you want it merged as is?

**Comment by jduckles** [comment]
@apawlik *nudge*

**Comment by apawlik** [comment]
@jduckles @gvwilson sorry I didn't get any notifications to my mailbox :-( Fixing now

**Comment by apawlik** [comment]
I rebased and then added your fixes. Even though I haven't touched the header in my blog post 'make site' fails with error I can't debug :-(

**Comment by gvwilson** [comment]
@apawlik are you opening a new one to take the place of this one?

**Comment by jduckles** [comment]
She opened a new one, it still had some cruft from other branches and I made a third one that I merged, thanks. It is live now.

**Pull request closed without merging**